### PR TITLE
Update facility selection layouts as per Zeplin

### PR DESCRIPTION
**Pull request by saket**
This PR will complete these pivotal stories:
- [Help Nurse find Facility she practices at post login](https://www.pivotaltracker.com/story/show/164135362)
- [Implement updated designs for facility selection screens](https://www.pivotaltracker.com/story/show/164138627)

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/facility/change/FacilityChangeScreen.kt [hunk 1]
```diff
@@ -6,6 +6,7 @@ import android.util.AttributeSet
 import android.view.View
```
Comment (saket) [inline app/src/main/java/org/simple/clinic/facility/change/FacilityChangeScreen.kt]: @sanchita the first commit copies a feature from `RegistrationFacilitySelectionScreen` to `FacilityChangeScreen` for showing the search field only when the facilities are available and shown on screen.

**Comment by saket** [comment]
@sanchita bump?

File: app/src/main/res/layout/list_facility_selection_progress.xml [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
+<androidx.cardview.widget.CardView style="@style/Clinic.FacilitySelection.CardView"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
 
-  <LinearLayout style="@style/Clinic.FacilitySelection.FacilityContainer">
+  <com.facebook.shimmer.ShimmerFrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    app:shimmer_auto_start="true"
+    app:shimmer_base_color="#c8cbcc"
```
Comment (sanchita) [inline app/src/main/res/layout/list_facility_selection_progress.xml]: What is this color hex?
Is this the shimmer color?

Comment (saket) [reply app/src/main/res/layout/list_facility_selection_progress.xml]: We made some changes to this layout: https://github.com/simpledotorg/simple-android/pull/523/commits/3713ec4ded91dff0447dc30c6b1cd99a1898ea53

**Pull request merged**